packages/app/src/version.ts

```diff
@@ -0,0 +1,2 @@
+// generated by genversion
+export default '10.8.1';
```

Comment:
generated files shouldn't be committed I don't think? may need to update .gitignore for new .ts version of file?

Reply:
You're correct, added to gitignore. Version `TS` files removed.